docroot/sites/its.uiowa.edu/modules/its_core/src/Plugin/Block/AlertTypeLegend.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\its_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides an 'Alert Type Legend' block.
+ *
+ * @Block(
+ *   id = "alert_type_legend_block",
+ *   admin_label = @Translation("Alert Type Legend"),
+ *   category = @Translation("Site custom")
+ * )
+ */
+class AlertTypeLegend extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return ['label_display' => FALSE];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $build['alert_type_legend'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'div',
+      '#attributes' => [
+        'class' => [''],
+      ],
+    ];
+    $build['alert_type_legend']['heading'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'div',
+      '#attributes' => [
+        'class' => [
+          'headline',
+          'headline--uppercase',
+          'h5',
+        ],
+      ],
+    ];
+
+    $build['alert_type_legend']['heading']['content'] = [
+      '#type' => 'html_tag',
+      '#tag' => 'span',
+      '#value' => 'Alert types',
+      '#attributes' => [
+        'class' => ['headline__heading'],
+      ],
+    ];
+    $build['alert_type_legend']['badges'] = [
+      'wrapper' => [
+        '#type' => 'markup',
+        '#markup' =>
+        '<div class="card__meta" >' .
```

Comment:
I'd also suggest that we remove the icons from displaying like we are doing on https://its.uiowa.ddev.site/alerts/calendar and remove the `card_meta` class here since this is not a card.    Preferably, this would display as a horizontal legend like the calendar page.    If we can't do that then maybe out put these in an `<ul>`?  `<ul class="element--list-none element--margin-none"><li><span class="badge badge--orange">Outage</span></li></ul>`

Reply:
If we did this and made it horizontal, where would we want to put it? at the bottom of the page? I guess it doesnt matter that much if its a block, but just for the intent

Reply:
noticed that though they are both horizontal and at the bottom, they are not the same visually. do we want to extend what was done to the calendar page (removing the views footer on the calendar view and replace with the block)?

Reply:
Went ahead and removed it in https://github.com/uiowa/uiowa/pull/7957/commits/746f7e2ed4c373da9b5fd7b0dcec3e962a90fc04. We can place the block in both locations when this is deployed.